test123.js

```diff
@@ -1 +1,10 @@
+// the hello world program
 console.log('Hello World');
+
+console.log('Hello Universe');
+
+console.log('Hello other stuff');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://release.k8s.dev.codacy.org/gh/troubleshoot-codacy/no-suggestions/pullRequest?prid=3394)
```suggestion
console.log("Hello other stuff");
```